scripts/deploy-to-cloud-run.sh

```
@@ -0,0 +1,136 @@
+#!/bin/bash
+set -e
+
+# Set project ID
+PROJECT_ID="fifiai"
+REGION="asia-southeast2"
+SERVICE_NAME="geo-tracking-service"
```

Shellcheck (0.10.0)

[warning] SERVICE_NAME appears unused. Verify use (or export if used externally).

(SC2034)